core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -21,6 +21,7 @@ import java.io.NotSerializableException
 import java.nio.ByteBuffer
 import java.util.concurrent.ConcurrentLinkedQueue
 
+import scala.collection.immutable.Map
```

Comment:
Can we use `immutable.Map` instead of `Map` in this file? It should really be `Map` and `mutable.*` for others.